static/src/javascripts/projects/commercial/modules/consentless/render-advert-label.ts

```diff
@@ -1,5 +1,44 @@
+/* eslint-disable @typescript-eslint/no-misused-promises
+-- Fastdom measure and mutate signatures are Promise<void>
+-- Nested fastdom measure-mutate promises throw the error:
+-- "Promise returned in function argument where a void return was expected"
+*/
+
+import crossIcon from 'svgs/icon/cross.svg';
 import fastdom from '../../../../lib/fastdom-promise';
-import { createAdLabel, shouldRenderLabel } from '../dfp/render-advert-label';
+import { shouldRenderLabel } from '../dfp/render-advert-label';
+
+export const createAdCloseDiv = (): HTMLElement => {
+	const closeDiv: HTMLElement = document.createElement('button');
+	closeDiv.className = 'ad-slot__close-button';
+	closeDiv.innerHTML = crossIcon.markup;
+	closeDiv.onclick = () => {
+		const container: HTMLElement | null = closeDiv.closest(
+			'.mobilesticky-container',
+		);
+		if (container) container.remove();
+	};
+	return closeDiv;
+};
+
+export const renderConsentlessAdvertLabel = (
+	adSlotNode: HTMLElement,
+): Promise<Promise<void>> => {
+	return fastdom.measure(() => {
```

Comment:
You can get rid of the eslint rule override at the top by doing something like:
```suggestion
	fastdom
		.measure(() => shouldRenderLabel(adSlotNode))
		.then((shouldRenderLabel) => {
			if (shouldRenderLabel) {
```